webui/react/src/pages/ExperimentDetails/ExperimentVisualization/HpTrialTable.tsx

```diff
@@ -103,7 +103,7 @@ const HpTrialTable: React.FC<Props> = ({
         if (isNumber(value) && isValidType) {
           return <HumanReadableFloat num={value} />;
         }
-        return record.hparams[key];
```

Comment:
the confusion comes from `boolean` being a valid `JSX.Element` while it's a value doesn't get rendered at all 🤔